conduct/code_of_conduct.md

```diff
@@ -1,131 +1,145 @@
-   expectations is preferable to demands for another person's time.
-
-7. **When we disagree, try to understand why**. Disagreements, both social and
-   technical, happen all the time and Pangeo is no exception.  Try to
-   understand where others are coming from, as seeing a question from their
-   viewpoint may help find a new path forward.  And don’t forget that it is
-   human to err: blaming each other doesn’t get us anywhere, while we can learn
-   from mistakes to find better solutions.
+## Consequences for Harassment and Discrimination
 
-8. **A simple apology can go a long way**. It can often de-escalate a situation,
-   and telling someone that you are sorry is an act of empathy that doesn’t
-   automatically imply an admission of guilt.
+If a member of the community at large engages in behavior which is harassing 
+or discriminatory in any way, once the incident is reported, 
+the [Code of Conduct Committee](mailto:TODO) may take any action they deem 
+appropriate listed in the [*Enforcement Manual*](enforcement.md). 
+These actions include, but are not limited to, issuing a warning, 
+requiring an apology, or a permanent or temporary ban from some 
+or all Pangeo spaces (mailing lists, gitter.im, etc.). 
 
 
-## Reporting
+## How to Report Harassment and Discrimination
 
 If you believe someone is violating the code of conduct, please report this in
 a timely manner. Code of conduct violations reduce the value of the community
 for everyone and we take them seriously.
 
 You can file a report by emailing
-[*TODO*](mailto:TODO) or by filing out
-[this form](TODO). For more details or
-information on reporting in-person at an event, please see our Reporting
-Guidelines.
-
-The online form gives you the option to keep your report anonymous or request
+[*TODO*](mailto:TODO) or by filing out 
+[this form](TODO). The online form gives you the option to keep your report anonymous or request
 that we follow up with you directly. While we cannot follow up on an anonymous
 report, we will take appropriate action.
 
-
-## Enforcement
-
+For more details or information on reporting in-person at an event, please see our Reporting
+Guidelines.   
 For information on enforcement, please view the [*Enforcement
 Manual*](enforcement.md).
 
-Original text courtesy of the [*Jupyter*](http://jupyter.org/conduct/) Project,
-modified by Pangeo.  We are grateful to those projects for contributing these
-materials under open licensing terms for us to easily reuse.
 
-All content on this page is licensed under a [*Creative Commons
-Attribution*](http://creativecommons.org/licenses/by/3.0/) license.
+## Periodic Review
+
+Pangeo strives to create a positive and inclusive environment. As such, the Pangeo **leadership team** (update with correct wording) commits to an annual review of the Code of Conduct to ensure that it continues to align with this goal and address the needs of our community.
```

Comment:
is this intended to be the steering council or another body?

Reply:
i believe so.